contracts/token/ERC20/MintableToken.sol

```diff
@@ -7,7 +7,6 @@ import "../../ownership/Ownable.sol";
 /**
  * @title Mintable token
  * @dev Simple ERC20 Token example, with mintable token creation
- * @dev Issue: * https://github.com/OpenZeppelin/openzeppelin-solidity/issues/120
```

Comment:
let's keep the link to the issue.

Reply:
Sorry here again, I didn't see Matt's comment. I should go to bed now :p scratch anything I said that you introduce because of a previous review.

Reply:
@elopio No worries! I'll update the PR in a while or tomorrow morning.

Thanks for the review.

Reply:
@elopio Actually, it was just a minor change, so I got it done rightaway.

Check it out when you get a chance. Thanks again!